src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/invalidjavadocposition/InputInvalidJavadocPosition.java

```diff
@@ -65,4 +65,26 @@ void method() {
         /** violation */ @Deprecated int variable3; // violation
     }
 }
+class GenericConstructor {
+    /** valid */
+    <E extends String> GenericConstructor(E a) {}
+
+    </** violation */E extends String>  GenericConstructor(E a, E b) {} // violation
+
+    <E extends String> /** violation */ GenericConstructor(E a, E b, E c) {} // violation
```

Comment:
To avoid confusion:
Violation -> invalid.

Reply:
@romani ping